CONST_Makefile

```diff
@@ -459,6 +459,7 @@ $(OUTPUT_DIR)/build.js: .build/node_modules.timestamp .build/build.js
 		bootstrap/dist/js/bootstrap.min.js \
 		proj4/dist/proj4.js \
 		d3/d3.min.js \
+		fuse.js/src/fuse.js \
```

Comment:
I've just seen that the fuse repo does include a minified version of the library: `fuse.js/src/fuse.min.js`. This is the one we should add to our build.